spaceship/lib/spaceship/portal/certificate.rb

```diff
@@ -298,16 +315,26 @@ def create!(csr: nil, bundle_id: nil)
 
           # look up the app_id by the bundle_id
           if bundle_id
-            app = portal_type.set_client(client).find(bundle_id)
-            raise "Could not find app with bundle id '#{bundle_id}'" unless app
-            app_id = app.app_id
+            if bundle_id.start_with?('merchant.')
+              merchant = Spaceship::Portal.merchant.find(bundle_id)
+              raise "Could not find merchant with bundle id '#{bundle_id}" unless merchant
```

Comment:
This this really also called `bundle id`, although it refers to a merchant? Or is this just how our internal variable is called?

Reply:
It's called `bundle_id`. Refer to this 
![Screenshot 2019-03-14 10 19 01](https://user-images.githubusercontent.com/2100166/54368689-9a2df600-4642-11e9-886c-9176dd9bd5b4.png)

`bundle_id` is `merchant.com.krausefx.app.production` which is what we are using in here.